<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyMeasureHoleFormClass</class>
 <widget class="GUIFormBase" name="PropertyMeasureHoleFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyMeasureHoleForm</string>
  </property>
  <widget class="QPushButton" name="pushButtonEditLibFolder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分類選択</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyMeasureHole.qrc">
     <normaloff>:/Qtres/Class.png</normaloff>:/Qtres/Class.png</iconset>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MinD</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MaxD</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelLibFolderName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ＸＸＸ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>登録されているライブラリリスト</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_89">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>最小許容径(μｍ)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMinDiameter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMaxDiameter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_90">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>最大許容径（μｍ）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_91">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>ノイズサイズドット</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditNoiseSize">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_92">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>探索ドット</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditSearchDot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditLibName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Library Name</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Mono">
     <widget class="QSpinBox" name="EditPickupH">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditPickupL">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_94">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>抽出輝度　明側</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_93">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>抽出輝度　暗側</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreateNewMono">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create new</string>
      </property>
      <property name="icon">
       <iconset resource="PropertyMeasureHole.qrc">
        <normaloff>:/Qtres/Make.png</normaloff>:/Qtres/Make.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdateMono">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset resource="PropertyMeasureHole.qrc">
        <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Color">
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset resource="PropertyMeasureHole.qrc">
        <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
      </property>
     </widget>
     <widget class="QFrame" name="frameColorCube">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreateNew">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create new</string>
      </property>
      <property name="icon">
       <iconset resource="PropertyMeasureHole.qrc">
        <normaloff>:/Qtres/Make.png</normaloff>:/Qtres/Make.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyMeasureHole.qrc"/>
 </resources>
 <connections/>
</ui>
